chapters/bizleg-case-studies/startups/yellr.md

```diff
@@ -0,0 +1,208 @@
+# yellr
+### report for your community 
+
+Table of Contents
+
+
+*Executive summary*
+
+*Background research*
+
+*Situational Analysis*
+
+*Objectives*
+
+*Action Plan*
+
+*Financial Data*
+
+## Executive Summary
+
+Who we are:
+
+    Yellr was created as a hacks/hackers project by a group of volunteer developers and journalists in association with WXXI.
+
+What we sell:
+
+    Yellr is a software platform and app for journalists to use to reach out and engage hyper-local communities by creating geo-fenced “assignments” which will notify users of questions that a journalist wishes to be answered by users. Users can anonymously post pictures, audio, videos, and text to WXXI, which can be curated by journalists to publish web stories.
+
+Who we sell to:
+
+Yellr specifically targets news organizations as our main customers. We plan on assisting these organizations in the process of marketing the app to local communities by developing a customized yellr marketing plan. 
+
+## Background Research and Industry Overview
+
+### How Journalism has evolved
+In 1983, 90% of the media was controlled by 50 companies, but by 2011 was consolidated to 6 corporations. The six corporations control news outlets and the delivery of news to the public through television, radio, social media, and more. 10% of news outlets operate 
+Social Media encourages citizen journalism 
```

Comment:
This paragraph seems a little disjointed. Is there a missing portion of this paragraph?